h/form.py

```diff
@@ -15,16 +20,29 @@
 )
 
 
-def init():
-    from deform import Form
-    from deform_jinja2 import jinja2_renderer_factory
-    from deform_jinja2.translator import PyramidTranslator
+# We supply our own Jinja2 renderer factory because deform_jinja2's one doesn't
+# support autoescaping.
+def jinja2_renderer_factory(config):
+    environment = []
+
+    def jinja2_renderer(template_name, **kwargs):
+        if not environment:
+            searchpath = [pkg_resources.resource_filename(*(path.split(':')))
+                          for path in SEARCH_PATHS]
+
+            environment.append(config.get_jinja2_environment().overlay(
+                loader=pyramid_jinja2.SmartAssetSpecLoader(
+                    searchpath=searchpath)))
+
+        if '.jinja2' not in template_name:
```

Comment:
For some reason it was written this way in `deform_jinja2`, but there doesn't seem to be any reason not to change it, done